packages/path_provider/path_provider_platform_interface/pubspec.yaml

```diff
@@ -3,20 +3,20 @@ description: A common platform interface for the path_provider plugin.
 homepage: https://github.com/flutter/plugins/tree/master/packages/path_provider/path_provider_platform_interface
 # NOTE: We strongly prefer non-breaking changes, even at the expense of a
 # less-clean API. See https://flutter.dev/go/platform-interface-breaking-changes
-version: 2.0.0-nullsafety
+version: 2.0.0
 
 dependencies:
   flutter:
     sdk: flutter
-  meta: ^1.3.0-nullsafety.3
-  platform: ^3.0.0-nullsafety.4
-  plugin_platform_interface: ^1.1.0-nullsafety
+  meta: ^1.3.0
+  platform: ^3.0.0
+  plugin_platform_interface: ">=1.0.0 <3.0.0"
```

Comment:
Hasn't 2.0.0 been published already? https://pub.dev/packages/plugin_platform_interface/versions/2.0.0

Reply:
Ah, this is to prevent a problem with plugins that are using ^1.1.0, gotcha!